content/en/docs/reference/kubernetes-api/ref/authentication/tokenrequest-v1.md

```diff
@@ -0,0 +1,161 @@
+---
+api_metadata:
+  apiVersion: "authentication.k8s.io/v1"
+  import: "k8s.io/api/authentication/v1"
+  kind: "TokenRequest"
+collapsible: false
+content_type: "api_reference"
+description: "TokenRequest requests a token for a given service account."
+draft: false
+title: "TokenRequest"
+weight: 2
+---
+
+`apiVersion: authentication.k8s.io/v1`
+
+`import "k8s.io/api/authentication/v1"`
+
+
+## TokenRequest
+
+TokenRequest requests a token for a given service account.
+
+
+- **apiVersion**: authentication.k8s.io/v1
+
+
+- **kind**: TokenRequest
+
+
+- **metadata** (<a href="{{< ref "../common-definitions/objectmeta-#objectmeta" >}}">ObjectMeta</a>)
+
+
+- **spec** (<a href="{{< ref "../authentication/tokenrequest-v1#tokenrequestspec" >}}">TokenRequestSpec</a>), required
+
+
+- **status** (<a href="{{< ref "../authentication/tokenrequest-v1#tokenrequeststatus" >}}">TokenRequestStatus</a>)
+
+
+
```

Comment:
nit: Are these extra lines required to layout the page?

Reply:
No, they are not required. I have to master go templating a little bit more to remove these empty lines